Dockerfile

```diff
@@ -99,22 +127,25 @@ RUN apt-get update && \
 
 COPY ./authentik/ /authentik
 COPY ./pyproject.toml /
+COPY ./poetry.lock /
 COPY ./schemas /schemas
 COPY ./locale /locale
 COPY ./tests /tests
 COPY ./manage.py /
 COPY ./blueprints /blueprints
 COPY ./lifecycle/ /lifecycle
-COPY --from=go-builder /work/bin/authentik /bin/authentik
+COPY --from=go-builder /go/authentik /bin/authentik
+COPY --from=python-deps /work/venv /venv
```

Comment:
I'm still not really a fan of copying an entire virtualenv into the container but if it makes the build faster and doesn't add (much) size I suppose I don't really have an argument against it